packages/components/src/LabeledInputSwitch.tsx

```diff
@@ -0,0 +1,23 @@
+import { InputSwitch } from '.';
+import { ReactElement } from 'react';
+
```

Comment:
That's the verbose way. 
Another way would be to define the Props type by extending the `InputSwitch` props. The passed through props could then be spreaded into the wrapped component with a one liner.